examples/ui-demo/src/app/providers.tsx

```diff
@@ -35,6 +36,8 @@ export const Providers = (props: PropsWithChildren<{}>) => {
     const primaryColor = config.ui.primaryColor[config.ui.theme]
     root?.style.setProperty("--akui-fg-accent-brand", primaryColor)
     root?.style.setProperty("--akui-btn-primary", primaryColor)
+  
+    root?.style.setProperty("--akui-border-radius-base", getBorderRadiusValue(config.ui.borderRadius))
```

Comment:
oh small nit: we export a function that generates the variable name for you too: `getColorVariableName` and `getBorderRadiusVariableName` or something along those lines. would be good to use those here in case something changes in the future